src/elements/content-sidebar/activity-feed/task-form/TaskForm.tsx

```diff
@@ -0,0 +1,525 @@
+import * as React from 'react';
+import noop from 'lodash/noop';
+import getProp from 'lodash/get';
+import isEmpty from 'lodash/isEmpty';
+import classNames from 'classnames';
+import { FormattedMessage, injectIntl } from 'react-intl';
+import type { IntlShape } from 'react-intl';
+
+import Form from '../../../../components/form-elements/form/Form';
+import ModalActions from '../../../../components/modal/ModalActions';
+import ContactDatalistItem from '../../../../components/contact-datalist-item/ContactDatalistItem';
+import TextArea from '../../../../components/text-area';
+import DatePicker from '../../../../components/date-picker/DatePicker';
+import Checkbox from '../../../../components/checkbox';
+import PillSelectorDropdown from '../../../../components/pill-selector-dropdown/PillSelectorDropdown';
+import Button from '../../../../components/button/Button';
+import PrimaryButton from '../../../../components/primary-button/PrimaryButton';
+import { ButtonType } from '../../../../components/button';
+import { TooltipPosition } from '../../../../components/tooltip';
+import { FeatureFlag } from '../../../common/feature-checking';
+
+import {
+    TASK_COMPLETION_RULE_ANY,
+    TASK_COMPLETION_RULE_ALL,
+    TASK_EDIT_MODE_CREATE,
+    TASK_EDIT_MODE_EDIT,
+} from '../../../../constants';
+import { ACTIVITY_TARGETS, INTERACTION_TARGET } from '../../../common/interactionTargets';
+
+import commonMessages from '../../../../common/messages';
+import messages from './messages';
+import commentFormMessages from '../comment-form/messages';
+import TaskError from './TaskError';
```

Comment:
move this `import TaskError from './TaskError';` import after the `PrimaryButton` import